### PR TITLE
[16.0][ADD] mrp_bom_image

### PR DESCRIPTION
**Pull request by quentinDupont**
New module :
- Product image is added to BoM form view.
- Module adds 3 image size to BoM : image (image_1024), image_512 and image_128.

image

mrp_bom_image_kanban

### CODE REVIEW

File: mrp_bom_image/models/mrp_bom.py [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+# @author: Quentin DUPONT (quentin.dupont@grap.coop)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from odoo import fields, models
+
+
+class MrpBom(models.Model):
+    _inherit = "mrp.bom"
+
+    image = fields.Binary(related="product_tmpl_id.image_1024")
```
Comment (legalsylvain) [inline mrp_bom_image/models/mrp_bom.py]: naive question : why not use image_1024 ? 

I mean : 

```
    image_1024 = fields.Binary(related="product_tmpl_id.image_1024")
```

**Review by legalsylvain — APPROVED** [review]
Code review / no test. LGTM !

File: mrp_bom_image/models/mrp_bom.py (hunk 1 above)
Comment (quentinDupont) [reply mrp_bom_image/models/mrp_bom.py]: seems like a mistake ;)
After a brief check, seems like image_1024 is the norm so i will change this !

Comment (quentinDupont) [reply mrp_bom_image/models/mrp_bom.py]: voilà :) :heavy_check_mark:

**Comment by luisDIXMIT** [comment]
Why can't i upload an image? Maybe i have to do something special before edit it?

@etobella

**Comment by quentinDupont** [comment]
> Why can't i upload an image? Maybe i have to do something special before edit it?
> 
> @etobella

You have to upload in Product form view and it will appear in BoM form :)

**Comment by legalsylvain** [comment]
@quentinDupont is it possible to modify the kanban view of the mrp.bom to display the new image you introduce ? 

thanks !

**Comment by quentinDupont** [comment]
> @quentinDupont is it possible to modify the kanban view of the mrp.bom to display the new image you introduce ?
> 
> thanks !

done !

**Comment by quentinDupont** [comment]
ping @OCA/manufacturing-maintainers if you want to review this one :)

File: mrp_bom_image/__manifest__.py [hunk 2]
```diff
@@ -0,0 +1,20 @@
+# Copyright (C) 2024 - Today: GRAP (http://www.grap.coop)
+# @author: Quentin Dupont (quentin.dupont@grap.coop)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+{
+    "name": "MRP BoM Image",
+    "summary": "Add product Images to BoM",
+    "version": "16.0.1.0.0",
+    "category": "GRAP - Custom",
```
Comment (grindtildeath) [inline mrp_bom_image/__manifest__.py]:
```suggestion
    "category": "Manufacture",
```

**Review by grindtildeath — APPROVED** [review]
LGTM, please fix category in manifest and we can merge

**Comment by quentinDupont** [comment]
> LGTM, please fix category in manifest and we can merge

Change done !
Make only 1 commit instead of 3 
Thanls for your review !

**Pull request merged**